lib/gbhw.fs

```diff
@@ -1,7 +1,7 @@
-require ./asm.fs
 also gb-assembler
 
 ( Special registers! )
+[host]
```

Comment:
I like it. Having to explicitly use `[host]` looks a bit funny now, but I'm assuming this will make more sense when we start "overriding" more words that work on the target.

Reply:
Exactly. We should need them less and less though. At the end, most words will do a little bit on the host and a bit on the target.

I am using the potential definition for colon in the target here again to get a feeling of how it should be.